modules/pstudioBidAdapter.js

```diff
@@ -417,7 +414,7 @@ function validateSizes(sizes) {
   );
 }
 
-function getBidFloor(bid) {
+/* function getBidFloor(bid) {
```

Comment:
Why are you dropping support for floors?

Reply:
Because we determine floor price on our internal SSP system based on ad tag id